lib/modules/uscore_v3.1.0/us_core_careteam_sequence.rb

```diff
@@ -236,9 +229,33 @@ def validate_resource_item(resource, property, value)
         @instance.save!
       end
 
-      test 'All references can be resolved' do
+      test 'The server returns expected results when parameters use composite-or' do
```

Comment:
Is it really just CareTeam and MedicationRequest that have multiple or?

Reply:
They're the only  ones with required searches